client/src/main/java/com/influxdb/client/TasksApi.java

```diff
@@ -266,6 +267,15 @@ Task createTaskEvery(@Nonnull final String name,
                          @Nullable final String userID,
                          @Nullable final String orgID);
 
+    /**
+     * Lists tasks, limit 100.
+     *
+     * @param query query params for task
+     * @return A list of tasks
+     */
+    @Nonnull
+    List<Task> findTasks(@NotNull TaskQuery query);
```

Comment:
Use `javax.annotation.Nonnull` annotation and add `final` modifier
```suggestion
    List<Task> findTasks(@Nonnull final TaskQuery query);
```